### PR TITLE
Update synthesizer/train.py, vocoder/train.py, synthesizer/synthesize.py, vocoder_preprocess.py to Allow Vocoder_preprocess to Work (Win10 GPU)

### PR DESCRIPTION
**Pull request by Tomcattwo**
Pull requested by @blue-fish 
@tomcattwo edited synthesizer/train.py per issue #669 and blue-fish/Real-Time-Voice-Cloning@89a9964 to fix Win10 pickle issue.
This will allow Windows users using a GPU to properly run synthesizer training.

### CODE REVIEW

**Comment by Tomcattwo** [comment]
Pulls requested by @blue-fish
1) In synthesizer/train.py @Tomcattwo edited per issue #669 and blue-fish/Real-Time-Voice-Cloning@89a9964 to fix Win10 pickle issue. This will allow Windows users using a GPU to properly run synthesizer training.
2) In vocoder/train.py @Tomcattwo edited per issue #669 and blue-fish/Real-Time-Voice-Cloning@89a9964 to fix Win10 pickle issue. This will allow Windows users using a GPU to properly run synthesizer training.
3) In synthesizer/synthesize.py, following edits made by @tomcattwo:
    a) added Win10 pickle fix per issue CorentinJ#669 and blue-fish/Real-Time-Voice-Cloning@89a9964 (lines 12 and 70)
    b) edited line 19 to delete "hparams" as argument; hparams_debug_string() needs 0 arguments. Allows vocoder_preprocess.py to run properly on WIn10/GPU (CUDA) systems)
    c) edited line 87 to fix improper definition for mels_out per @blue-fish recommendation in issue#729 and issue CorentinJ#833; Allows vocoder_preprocess.py to run properly on WIn10/GPU (CUDA) systems)
4) In vocoder_preorocess.py @Tomcattwo changed line 47 per recommendation by @blue-fish in issue CorentinJ#833; allows --cpu argument to be properly recognized

File: synthesizer/synthesize.py [hunk 1]
```diff
@@ -8,13 +8,15 @@
 import numpy as np
 from pathlib import Path
 from tqdm import tqdm
-
+#@tomcattwo added below line to fix Win10 pickle error per issue #669 and  blue-fish/Real-Time-Voice-Cloning@89a9964
```
Comment (blue-fish) [inline synthesizer/synthesize.py:11]: Please remove all new comments. This information will be summarized in the commit message.

**Review by blue-fish — CHANGES_REQUESTED** [review]
Thank you for submitting the pull request @Tomcattwo .

Please remove all new comments. This information will be summarized in the commit message.

**Comment by Tomcattwo** [comment]
Additional comments removed. Please review.

**Review by blue-fish — APPROVED** [review]

**Comment by blue-fish** [comment]
Resolves issues reported in #669 and #833. Also readme update to address #728.

**Review by CorentinJ — APPROVED** [review]

**Comment by blue-fish** [comment]
Thank you for submitting the pull request @Tomcattwo .

**Pull request merged**